airbyte-connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -189,11 +189,15 @@ components:
       type: object
       required:
         - manifest
+        - config
       properties:
         manifest:
           type: object
           description: The config-based connector manifest contents
           # $ref: "#/components/schemas/ConnectorManifest"
+        config:
```

Comment:
huh, interesting. did you need this for wikipedia or is it just precautionary? I wonder what this means for the UI -- is the config now required for things other than the `test` button?

Reply:
Not for wikipedia specifically, but a pattern that i would say does come up more than you would think. Any partner api that uses a custom hostname like Sentry or Zendesk would run into this. 

Lake and I discussed this a little bit. It might be a little wonky since its more than likely that a developer would configure their config before testing so the URLs might not look quite right at the beginning. But at least if we attempt to interpolate w/ the config, that's our best effort to display a valid URL. The bare minimum we want is that this request doesn't crash or return an error since that would be more annoying to the user to get that warning every time they modify their YAML.

Reply:
Yeah, basically the best UX would be, say, the user writes out a manifest that defines some stream's URL as `https://api.com/{{config['location']}}`. Then this endpoint takes in the manifest and the config. It tries to resolve the interpolated string, so if the config contains a `location` (say, `california`) then it will insert that location into the returned URL, i.e. returning `https://api.com/california` for that stream.

But, if the config does _not_ contain a `location`, then this endpoint should just return the string without resolving the interpolated value, i.e. returning `https://api.com/{{config['location']}}`